### ISSUE TITLE
localhost:port/swagger/index.html is slow to use and times out for a 

### ISSUE THREAD

**Issue by wvmstr** [RicoSuter/NSwag#3235]
Just getting started with swagger and NET5.0

The (open) api works fine directly from browser or Postman.

The swagger/index.html feels like it is bogged down by something just to get to the point where I can enter an Id. After 20 sec or so the requested record is returned with a 200
A GetAll against a single table with about 1000 records times out 3 or 4 times before it returns the records, taking several minutes to get to this point.

VS2019 16.8.3 no debugger
NSwag 13.9.4
Swashbuckle 5.6.3

Not sure what kind of performance to expect.

**Comment by jeremyVignelles** [comment]
Why are you using NSwag for? You're saying that you're using Swashbuckle, these are 2 different things.

> A GetAll against a single table with about 1000 records times out 3 or 4 times before it returns the records, taking several minutes to get to this point.

I assume that you're saying that the Swagger UI is slow when there is a lot of data to display. I've already seen that, but it's [another project](https://github.com/swagger-api/swagger-ui)

**Comment by wvmstr** [comment]
Just getting started so I am not sure what is what functionality wise.

Get /api/Transaction
Try it out
Click on Execute


After 10 seconds loading shows up
Then 3 or 4 times "the page is not responding" and click WAIT, then the data shows up.

Are you telling me this is not a good tool to work against a production db?

**Comment by jeremyVignelles** [comment]
No, I'm saying that the bug you're describing is a frontend bug caused by swaggerUI, which is a different tool.

**Comment by wvmstr** [comment]
Thanks for clarifying I am using swagger UI.

I will repost to that project.
